### PR TITLE
#163519156 Users should be able to bookmark articles for reading later

### PR DESCRIPTION
**Pull request by NkFab**
#### What does this PR do?

Have the feature of bookmarking an article or removing an article from bookmarks functioning.

#### Description of Task to be completed?

Have following endpoints working properly: 
`POST api/v1/articles/:articleSlug/bookmark`
`DELETE api/v1/articles/:articleSlug/bookmark`

#### How should this be manually tested?

Clone the repository, in the terminal run the following command `npm install` or `yarn` to install dependencies the after the installation was successful run `npm run migrate:seed` or `yarn migrate:seed`. When the migrations are installed run `npm run dev` or `yarn dev`, by the use of ***Postman*** test the endpoints described.  

#### What are the relevant pivotal tracker stories?

#163519156

### CODE REVIEW

**Review by Nziranziza — COMMENTED** [review]
Did you remember to change the res.body message from the last demo?

**Review by Nziranziza — COMMENTED** [review]
Did you remember to change res.body.message as our PO said in the last demo call?

File: __tests__/routes/likeComment.test.js [hunk 1]
```diff
@@ -35,11 +35,11 @@ describe('likeComment', () => {
     await Comment.destroy({ where: { id: testComment.id } });
   });
 
-  test('should like a comment', async (done) => {
+  test('should like a comment', async done => {
     const res = await request(app)
       .post(`${urlPrefix}/articles/${testArticle.slug}/comments/${testComment.id}/like`)
       .set('authorization', testToken);
-
+    console.log(res, res.body, testComment.id, '////////////////////////////', testArticle);
```
[the remark this answers is not in the document]
Comment (NkFab) [reply __tests__/routes/likeComment.test.js]: Removed

File: __tests__/routes/article.test.js [hunk 2]
```diff
@@ -329,4 +330,62 @@ describe('articles', () => {
...
+    expect(res.body.status).toBe(401);
+    expect(res.body.message).toBe('No auth token');
+  });
+
+  test('Should return bookmark', async () => {
+    await Bookmark.destroy({ where: { userId: loginUser1.id, articleId: testArticle.id } });
+    const res = await request(app)
+      .delete(`${urlPrefix}/articles/${testArticle.slug}/bookmark`)
+      .set('Authorization', loginUser1.token);
+    console.log(res.body);
```
[the remark this answers is not in the document]
Comment (NkFab) [reply __tests__/routes/article.test.js]: Removed

**Review by Nziranziza — APPROVED** [review]

**Review by oesukam — APPROVED** [review]
@NkFab I have reviewed and approved

**Pull request merged**